http_logs/test_procedures/grpc/grpc-schedule.json

```diff
@@ -0,0 +1,56 @@
+{
+    "operation": "delete-index"
+},
+{
+    "operation": {
+        "operation-type": "create-index",
+        "settings": {{index_settings | default(default_index_settings | default({})) | tojson}}
+    }
+},
+{
+    "name": "check-cluster-health",
```

Comment:
We should embed the common_operations like this https://github.com/opensearch-project/opensearch-benchmark-workloads/blob/3d53dae7096eeb3c281c3ef15e3744adcb3d60f2/http_logs/test_procedures/default.json#L6-L8